src/api/v0.rs

```diff
@@ -551,14 +532,13 @@ pub fn modules_info_default(modules: &str, db: State<DBPool>) -> CORS<JSON<Modul
 /// }
 /// ```
 ///
-pub fn modules_info(modules: &str, db: State<DBPool>, offset: i64, limit: i64) -> CORS<JSON<ModulesInfoResponse>> {
-    info!("/modules/info/"; "modules" => modules, "offset" => offset, "limit" => limit);
+#[get("/modules/info/<modules>", rank=2)]
+pub fn modules_info(modules: &str, db: State<DBPool>) -> CORS<JSON<ModulesInfoResponse>> {
+    info!("/modules/info/"; "modules" => modules);
     let modules: Vec<&str> = modules.split(",").collect();
-    let result = get_groups_deps_vec(&db.conn(), &modules, offset, limit);
+    let result = get_groups_deps_vec(&db.conn(), &modules, 0, i64::max_value());
```

Comment:
This one I can more see keeping the offset/limit arguments on, but it's worth thinking about.  This is the only caller of this function right now.